### ISSUE TITLE
Error when installing newest version

### ISSUE THREAD

**Issue by ericdsw** [benevbright/react-navigation-collapsible#185]
## Information
- react-native version:  0.63.4
- react-navigation version: 5.15.12
- react-navigation-collapsible version: 5.10.1
- Platform (iOS/Android): Both
- react-native init or Expo: react-native

## Detail

Hey, I just submitted the pull request #183 which added a new `disableOpacity` configuration, which was published a few hours ago.

I just tried to update the library to the latest version by running `yarn upgrade react-navigation-collapsible --latest`, but it was raising the following error:

```
Exit code: 1
Command: cd example && yarn && cd .. && yarn tsc
Arguments:
Directory: /Users/erickdesedas/Development/projects/ASSA/health-app-customer/node_modules/react-navigation-collapsible
Output:
/bin/sh: line 0: cd: example: No such file or directory
```

Trying to uninstall and install it again throws the same error.

Now, checking the changes for the latest version I see this in the `package.json` file:

![image](https://user-images.githubusercontent.com/3594423/124945005-66c1c300-dfd3-11eb-8ade-f4e91261f502.png)

Which appears to be causing the issue with "postinstall", as the `example` directory is not present in the published version.

**Comment by benevbright** [comment]
Oops.... yeap. my mistake. on it. Thanks again @ericdsw

**Comment by benevbright** [comment]
fixed on `v5.10.2`. Thanks!!

**Comment by ericdsw** [comment]
Perfect, thanks!
Amazing library btw
